src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars--0 stars">
```

Comment:
```suggestion
    <div class="stars stars--0">
```
It's not critical, but I think it's more logical to set the default class name first, and then the modifier